modifiers/implicit_predicate_abstractor.h

```diff
@@ -0,0 +1,94 @@
+/*********************                                                  */
+/*! \file implicit_predicate_abstractor.h
+** \verbatim
+** Top contributors (to current version):
+**   Makai Mann
+** This file is part of the pono project.
+** Copyright (c) 2019 by the authors listed in the file AUTHORS
+** in the top-level source directory) and their institutional affiliations.
+** All rights reserved.  See the file LICENSE in the top-level source
+** directory for licensing information.\endverbatim
+**
+** \brief Implicit predicate abstraction based on
+**        Abstract Model Checking without Computing the Abstraction
+**        Stefano Tonetta
+**
+**/
+
+#pragma once
+
+#include "abstractor.h"
+
+namespace pono {
+
+class ImplicitPredicateAbstractor : public Abstractor
+{
+ public:
+  ImplicitPredicateAbstractor(const TransitionSystem & conc_ts,
+                              TransitionSystem & abs_ts)
+      : Abstractor(conc_ts, abs_ts),
```

Comment:
why not move the implementation to its cpp file?

Reply:
Good point -- changed in 54e716aabee3cff707cfdc5a22c601530234616a